scripts/append-boost-config-macros.cmake

```diff
@@ -0,0 +1,33 @@
+# run at the boost source root
```

Comment:
`cmake_minimum_required` needed in scripts too:
* http://cgold.readthedocs.io/en/latest/tutorials/cmake-sources/scripts.html#minimum-required-bad

Reply:
Ok